push-and-sign-artifact/action.yaml

```diff
@@ -0,0 +1,101 @@
+name: "Push and sign an artifact"
+author: "Robert Schardt <robert.schardt@greenbone.net>"
+description: "An action that pushes an artifact with oras and signs it with cosign"
+
+inputs:
+  artifact-file:
+    type: string
+    required: true
+  artifact-url:
+    type: string
+    description: "The artifact-url to upload the artifact into (with tag)."
+    required: true
+  artifact-folder:
+    type: string
+    description: "The path to the system folder to save the artifact file. Default is /tmp."
+    default: '/tmp'
+  registry-user:
+    type: string
+    description: "Registry user name."
+  registry-domain:
+    type: string
+    description: "Registry domain."
+  registry-token:
+    type: string
+    description: "Registry user token. Set is input only if you want to upload an artifact."
```

Comment:
Let's make them required other wise this action makes no sense or? Sorry my mistake.